src/vs/workbench/contrib/terminal/test/browser/xterm/decorationAddon.test.ts

```diff
@@ -21,24 +21,24 @@ import { ILifecycleService } from 'vs/workbench/services/lifecycle/common/lifecy
 import { TestLifecycleService } from 'vs/workbench/test/browser/workbenchTestServices';
 import { importAMDNodeModule } from 'vs/amdX';
 
-suite('DecorationAddon', async () => {
+suite('DecorationAddon', () => {
+	let decorationAddon: DecorationAddon;
+	let xterm: any;
```

Comment:
@meganrogge and @Tyriar I know this `any` is not ideal, but having the `async` suite here causes the async code to run while other tests are running and can interfere with the other tests.

Reply:
Change looks good, you should be able to import the type like this still though:

https://github.com/microsoft/vscode/blob/0350457d9961c4ac8a535121a02ca511fea96156/src/vs/workbench/contrib/terminal/browser/xterm/xtermTerminal.ts#L6